src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -79,6 +79,9 @@ const GuideAnchor = createReactClass({
   },
 });
 
+export const ConditionalGuideAnchor = ({condition, target, type, children}) =>
```

Comment:
What's the benefit of this over the (I think simpler) `condition && <GuideAnchor ...>`?

Reply:
The benefits are definitely more hypothetical. The primary reason I'd done the wrapping was a future world where we might put the pings *behind* the UI element, so the user can still click on the element itself mid-tour.

It's obviously not a vital feature, but I don't see much downside to `ConditionalGuideAnchor` except perhaps, we'd like to fold this into the `GuideAnchor` class itself.

Reply:
The advantages of wrapping (versus inserting) is that we have greater control of the actual element we're trying to focus and (for instance) can capture and pass down click events (whereas adjacent elements aren't aware of the target element).

We're not leveraging this now, but it feels better as a pattern.